CONTRIBUTING.md

```diff
@@ -114,10 +114,10 @@ included in the next released version.
 ## Building Gitea
 
 Generally, the go build tools are installed as-needed in the `Makefile`.
-An exception are the tools to build the CSS and images.
+An exception are the tools to build the CSS, JS and images.
 
-- To build CSS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 8.0 or above
-  with `npm` and then run `npm install` and `make css`.
+- To build CSS and JS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 10.0 or above
+  with `npm` and then run `npm install`, `make css` and `make js`.
```

Comment:
`make js` what's means?

Reply:
`make js` is a command that generates the javascript for the application